samples/Sentry.Samples.Console.Basic/Sentry.Samples.Console.Basic.csproj

```diff
@@ -4,8 +4,8 @@
     <OutputType>Exe</OutputType>
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
-    <TargetFramework>net8.0</TargetFramework>
-    <PublishAot>true</PublishAot>
+    <TargetFrameworks>net8.0;net6.0;netstandard2.1;netstandard2.0;net462</TargetFrameworks>
```

Comment:
Was this a mistake? netstandard on an executable doesn't make sense to me

Reply:
I've removed those on the feat/4.0.0 branch